Server/Services/ISSStationService.js

```diff
@@ -0,0 +1,60 @@
+const { cp } = require('fs');
+const http  = require('http');
+const firebaseFunctions = require('../firebaseFunctions');
+const url = 'http://api.open-notify.org/iss-now.json';
+const gap = 1000.0
+const heartRadius = 6374
+
+module.exports = {
+    checkISSPosition: function(res, uid) {
+        firebaseFunctions.getDataFromFireBase(uid, 'OpenMeteoService')
```

Comment:
Rather create an IssStation category in the DB instead of using openMeteoService. The user can use issService without having used openMeteoService